.github/workflows/shadow-repo-update.yml

```diff
@@ -69,3 +69,21 @@ jobs:
           author_email: sass.bot.beep.boop@gmail.com
           message: Update from https://github.com/${{ github.repository }}/commit/${{ github.sha }}
           commit: --allow-empty
+
+  diff-link:
```

Comment:
If you make this run after the push command, you can get rid of the "(may take a few minutes)" text since the comment will only post once the diff is actually visible.

Reply:
I was originally thinking that, but the push runs each time the branch is pushed to while the comment is only made when the PR is open, so I don't think they can be linked together that way